index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/styles.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="header-item" id="logo-container">
+      <label>John Doe</label>
+    </div>
+      <button class="header-item" id="menu-button">
+        <img src="icons/Icon-Menu.png" alt="btn">
+      </button>
+  </header>
```

Comment:
- [ ] Please know that the logo needs to be a link an `a` tag, it should lead to the header or to the website itself, it's up to you what to put in that link.
- [ ] Please know that the menu button should be inside a `nav` tag, as that will serve as your navigation menu later. 😁
- [ ] Please know that the src for the image is wrong, the folder is named `Icons` with a capital I kindly fix that, it's also recommended to use small letters for folder and file names so that you don't run into these issues. 😁